packages/engine-core/src/data-proxy/utils/getClientVersion.ts

```diff
@@ -36,7 +36,18 @@ async function _getClientVersion(config: EngineConfig) {
     const pkgURL = prismaPkgURL(`<=${major}.${minor}.${patch}`)
     const res = await request(pkgURL, { clientVersion })
 
-    return (await res.json())['version'] as string
+    const bodyAsText = res.text()
```

Comment:
One more question: shouldn't this be 
```suggestion
    const bodyAsText = await res.text()
```
?

Reply:
@aqrln Not because it's not node-fetch here, we are stealing types from it 😄 but in our implementation we don't need to await `text()`.